src/components/ChatApp.css

```diff
@@ -137,7 +170,10 @@ header{
     top: 16px;
     border-color: transparent transparent #d3e2f4 #d3e2f4;
     background: #d3e2f4;
-
+}
+.dark .message-container.SUSI::after{
```

Comment:
instead of using the CSS variables we can have sass variables in a sass file. which can then be used to make different themes for SUSI what do you think? same for the rest of the CSS